backend/.env.example

```diff
@@ -2,4 +2,4 @@ DJANGO_SETTINGS_MODULE={{project_name}}.settings.local
 CELERY_BROKER_URL=amqp://broker:5672//
 # Please choose postgres or sqlite as your DB:
 # DATABASE_URL=postgres://{{project_name}}:password@db:5432/{{project_name}}
-# DATABASE_URL=sqlite:///backend/db.sqlite3
+# DATABASE_URL=sqlite:///db.sqlite3
```

Comment:
In the README.md file we instruct people to run the application from within the backend directory, so the extra `backend` here would create a backend dir within backend.